packages/visualizations/app/styles/navi-visualizations/visualizations/table.less

```diff
@@ -219,3 +224,50 @@
     font-size: @font-size-xsmall;
   }
 }
+
+.table-header-cell__dropdown-container {
+  border: 1px solid @navi-gray-300;
+  box-shadow: 0 3px 8px fade(@navi-black, 38%);
+  font-size: @font-size-base;
+  margin-top: 20px;
+  padding: 10px;
+  max-width: 280px;
+
+  &:before, &:after {
+    border: solid transparent;
+    content: " ";
+    height: 0;
+    position: absolute;
+    pointer-events: none;
+    right: 8px;
+    top: -25px;
+    width: 0;
+  }
+
+  &:before {
+    border-bottom-color: @navi-gray-300;
+    border-width: 13px;
+  }
+
+  &:after {
+    border-bottom-color: @navi-white;
+    border-width: 14px;
+  }
+
+  .number-format-selector__section-header {
+    font-weight: 600;
+  }
+
+  .number-format-selector__section {
+    border-bottom: 0;
+    margin-bottom: 10px;
+    padding: 0;
+  }
+
+  .number-format-selector__format-input {
+    font-size: @font-size-base;
+    background-color: @navi-purple-lightest;
```

Comment:
We have been going with the denali styles nowadays. Could we use a light blue instead like in the mock?